app/views/sign_requests/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<% provide(:title, "Request signs") %>
+
+<div class="grid-x list">
+  <div class="list__section cell grid-x">
+    <div class="cell small-12 large-9 margin-bottom-2">
+      <h1>Request signs</h1>
+    </div>
+
+    <div class="cell small-12 large-6">
+      <h3 class="margin-bottom-1">Request signs on Facebook</h3>
+
+      <p>
+        Intro and overview information on Facebook requests, and possible future integration into this website.
+        Intro and overview information on Facebook requests, and possible future integration into this website.
+        Intro and overview information on Facebook requests, and possible future integration into this website.
+        Intro and overview information on Facebook requests, and possible future integration into this website.
+      </p>
+    </div>
+
+    <div class="cell small-12 large-6">
+      <div class="row align-right align-middle">
+        <%= inline_svg_pack_tag "media/images/facebook-light.svg", class: "margin-right-1", aria: true, title: "Facebook Logo" %>
```

Comment:
Visually, this looks quite out of place, though I can see why you've grabbed it. I believe the iconset we are using for this project has a Facebook icon that is probably better to use here, since it'll fit in with the rest of the visual style. I've also noticed that the icon height does not match the button height.

Reply:
This is the facebook icon that was already in the app, just recoloured using variables that were already there. I can investigate though, what icon set are we using in this app so I can have a look?

Reply:
I think Karyn has a icon idea to use here